hack/operator-release.sh

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o pipefail
+set -o errexit
+
+UNRELEASED_SEMVER="99.0.0-unreleased"
+PROJECT_ROOT="$(readlink -e "$(dirname "${BASH_SOURCE[0]}")"/../)"
+MANIFESTS_DIR="${PROJECT_ROOT}/deploy/olm-catalog/manifests"
+CSV="assisted-service-operator.clusterserviceversion.yaml"
+OPERATOR_VERSION=${OPERATOR_VERSION:-}
+
+COMMUNITY_OPERATORS_REPO="operator-framework/community-operators"
```

Comment:
The readMe file from https://github.com/operator-framework/community-operators says the repo is obsolete now. The operators from `operator-framework/community-operators/community-operators` are moved to a new location `redhat-openshift-ecosystem/community-operators-prod/operators`